test/integration/org/pih/warehouse/inventory/InventoryServiceTests.groovy

```diff
@@ -63,11 +64,11 @@ class InventoryServiceTests extends GroovyTestCase {
         supplierLocationType = LocationType.get(Constants.SUPPLIER_LOCATION_TYPE_ID)
 
         // get or create a default location
-        acmeLocation = DbHelper.creatLocationIfNotExist("Acme Supply Company", supplierLocationType)
+        acmeLocation = DbHelper.getOrCreateLocation('Acme Supply Company', supplierLocationType)
```

Comment:
For future reference, I prefer (and have tried to use) the convention findOrCreate since it matches what is available in later versions of Grails.

Reply:
That makes a lot of sense. Resolved in `b28afe136`